backend/app/utils/exceptions/logger.py

```
@@ -0,0 +1,76 @@
+    """
+    Logger Utility to standardize log format and include additional information.
+    """
+
+    def log(
+        self,
+        request_id,
+        correlation_id,
+        request,
+        response,
+        duration_ms,
+        extras={},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)